src/Verify/VerifySettings.cs

```diff
@@ -88,6 +89,14 @@ public void UseTextForParameters(string parametersText)
         this.parametersText = parametersText;
     }
 
+    public void UseUtf8NoBom() =>
```

Comment:
I dont see people wanting an instance API for this? wouldnt they only want the static API?

Reply:
Not sure. I want the static API indeed but I didn't know the criteria used for one or the other. Let me know and I can remove it.

Reply:
the instance API would only be if someone wants to mix BOM and non-bom in the same project. i dont think that would eve happen. so lets go with only the static API

Reply:
sounds good!

Reply:
done.